### PR TITLE
Added initial files

### PR DESCRIPTION
**Pull request by fred-jan**

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+filter:
+    excluded_paths: [tests/*]
+build:
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --coverage-clover=coverage.clover'
```
Comment (Feijs) [inline .scrutinizer.yml:7]: Can you fetch a test success/failure status from scrutinizer? If not, I would suggest adding a `.travis.yml` file
Edit: would add it anyway, to run the testsuite for both `5.5.9` and `7.0`

File: src/Gateway.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    public function initialize(array $parameters = array())
+    {
+        parent::initialize($parameters);
+
+        $this->parameters->set(
+            'connector',
+            Connector::create(
+                $this->getParameter('merchant_id'),
+                $this->getParameter('secret'),
+                $this->getTestMode() ? ConnectorInterface::EU_TEST_BASE_URL : ConnectorInterface::EU_BASE_URL
```
Comment (Feijs) [inline src/Gateway.php]: This limits the package to Europe.  EU can be default, but I would suggest an additional parameter to allow NA URLs.

File: src/ItemInterface.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout;
+
+interface ItemInterface extends \Omnipay\Common\ItemInterface
+{
+    /**
+     * @return int
```
Comment (Feijs) [inline src/ItemInterface.php]: Would prefer an additional description for these methods, similar to the original interface. Is this the amount of tax applied or the base amount over which tax is applied?

File: src/Gateway.php (hunk 2 above)
Comment (fred-jan) [reply src/Gateway.php]: Good point, I'll add the North America URL's as well.

I am actually considering to discard the `kco_rest` package and talk to the HTTP API directly using Guzzle. This seems to be the preferred way of implementing new Omnipay gateways: http://omnipay.thephpleague.com/gateways/build-your-own/.

File: .scrutinizer.yml (hunk 1 above)
Comment (fred-jan) [reply .scrutinizer.yml:7]: Scrutinizer already gives a success/failure status right?:
image

I will look into the possibility of having Scrutinizer build upon multiple PHP versions.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+language: php
+php:
+  - '5.5'
+  - '5.6'
+  - '7.0'
```
Comment (Feijs) [inline .travis.yml:5]: This should include
```
before_script:
  - composer install --prefer-source
```

Comment (fred-jan) [reply .travis.yml:5]: Yep, was still in the process of adding it (sadly Travis has no way to test configs without committing them to the repo), should work now.

File: .scrutinizer.yml (hunk 1 above)
Comment (fred-jan) [reply .scrutinizer.yml:7]: Since Scrutinizer doesn't support this, the separate PHP version checks are now performed using Travis.

**Pull request merged**